docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md

```diff
@@ -292,20 +292,50 @@ use Spryker\Zed\TaxAppExtension\Dependency\Plugin\CalculableObjectTaxAppExpander
 class CalculableObjectCustomerWithVertexCodeExpanderPlugin extends AbstractPlugin implements CalculableObjectTaxAppExpanderPluginInterface
 {
     /**
-     * @param \Generated\Shared\Transfer\CalculableObjectTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\CalculableObjectTransfer $calculableObjectTransfer
      *
      * @return \Generated\Shared\Transfer\CalculableObjectTransfer
      */
-    public function expand(CalculableObjectTransfer $quoteTransfer): CalculableObjectTransfer
+    public function expand(CalculableObjectTransfer $calculableObjectTransfer): CalculableObjectTransfer
     {
-        $quoteTransfer->getTaxMetadata()->setCustomer(['customerCode' => 'vertex-customer-code']);
+        $calculableObjectTransfer->getTaxMetadata()->setCustomer(['customerCode' => ['classCode' => 'vertex-customer-code']]);
+
+        return $calculableObjectTransfer;
+    }
+}
+```
+
+#### Configure Customer Exemption Certificate Expander plugins
+
+For order:
+
+`Pyz/Zed/{YourDesiredModule}/Communication/Plugin/Order/OrderCustomerWithVertexExemptionCertificateExpanderPlugin.php`
+
+and for quote:
+
+`Pyz/Zed/{YourDesiredModule}/Communication/Plugin/Quote/CalculableObjectCustomerWithVertexExemptionCertificateExpanderPlugin.php`
+
+The following code sample shows how to introduce the following expander plugins:
```

Comment:
If you want to mention the plugins above, then rephrase this sentence as follows:
```suggestion
The following code sample shows how to implement the previously mentioned plugins:
```